test/TesterAzureUtils/AzureGossipTableTests.cs

```diff
@@ -26,7 +27,8 @@ public class AzureGossipTableTests : AzureStorageBasicTests
 
         public AzureGossipTableTests()
         {
-            logger = LogManager.GetLogger("AzureGossipTableTests", LoggerType.Application);
+            var loggerFactory = TestingUtils.CreateDefaultLoggerFactory($"{this.GetType().Name}.log");
```

Comment:
implement IDisposable and dispose the factory after the test is done (or just pass the null factory instead).